trade-finance-manager-api/src/constants/payloads.js

```diff
@@ -24,10 +24,11 @@ const PORTAL = {
 
 const TFM = {
   USER: {
+    azureOid: String,
     username: String,
     email: String,
-    salt: String,
-    hash: String,
+    // salt: String,
+    // hash: String,
```

Comment:
Can remove these?

Reply:
Yes, we will need to cleanup and remove `password`, `salt`, `hash` functionality from TFM.

Unless we will need to add Feature Flag for new functionality and keep old one also working.

Reply:
Comments were removed.